### ISSUE TITLE
For accessibility, the session should be extendable

### ISSUE THREAD

**Issue by joelanman** [alphagov/govwifi-admin#1410]
https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-required-behaviors.html

The best way to meet this criteria is probably to have a popup to extend the session, which is currently 30 mins. We might need to add this to our accessibility statement in the meantime

**Comment by joelanman** [comment]
thanks! What was the solution?

**Comment by A1Robson** [comment]
Hi Joe,
Sorry to give the impression it's been resolved - the issue has just been
moved across into our Jira stack so we can track it in one place.

Regards

Aaron
